librz/core/disasm.c

```diff
@@ -5397,7 +5397,9 @@ RZ_API int rz_core_print_disasm(RzPrint *p, RzCore *core, ut64 addr, ut8 *buf, i
 				ds_print_core_vmode(ds, ds->asm_hint_pos);
 			}
 		}
-		ds_print_op_size(ds);
+		if (!mi_found || mi_type != RZ_META_TYPE_STRING) {
```

Comment:
does it only happen with strings? what about structs? types? and other things that potentially have a length displayed?

Reply:
I see also `data` meta display the length, but in that case it is showed in a nicer way. I'm not aware of other cases, are you?

Reply:
I did not check. but I won't be surprised if other places have this duplication as well.